Added path details to search item in FileSearch

MarcoSennHaag opened the pull request:
Added path details to search item in FileSearch (#1165)

Please review also on a Mac if possible.

oliverschwendener commented:
Can you please separate the bugfix and the added feature into separate pull requests, as stated in the contribution guideline: https://github.com/oliverschwendener/ueli/blob/main/CONTRIBUTING.md#pull-requests

MarcoSennHaag commented:
> Please separate the bugfix and the added feature into separate pull requests, as stated in the contribution guideline: https://github.com/oliverschwendener/ueli/blob/main/CONTRIBUTING.md#pull-requests

Splitted as requested.

MarcoSennHaag commented:
Do I need to update this PR once #1356 is merged to the main branch?

oliverschwendener commented:
Please have a look at #1359.

I added your suggested change there. Additionally I added the new `details` property to all extensions where it makes sense. I think it makes sense if we close this PR in favor of the other PR.

MarcoSennHaag commented:
> Please have a look at #1359.
> 
> I added your suggested change there. Additionally I added the new `details` property to all extensions where it makes sense. I think it makes sense if we close this PR in favor of the other PR.

I agree with you. I will close this pull request in favor of your change.
